js/src/stringCollector.js

```diff
@@ -102,7 +136,7 @@ export class StringCollector {
     // Process other tags
     switch (tag) {
       case 'br':
-        this.processText()
+        this.processText(false)
```

Comment:
processText(true) - if the trimEndSpaces=true is only a specific case for the br tag, maybe we should keep the original logic in the 'processText' method and add a new method which is very special for the br tag to make the original behaviour remain the same?
Or the original method might be split into smaller pieces of code which might be reused in the new method which is specific for the `br` tag.
My concern is that the code of `processText` is getting huge

Reply:
I will split up the functionality into two functions.